app/controllers/dashboards_controller.rb

```diff
@@ -54,6 +55,12 @@ def push
 
   private
 
+  def verify_buildable
+    return true if Version.buildable?
+    flash.alert = 'Cannot build a new version since no new uploads have been made.'
```

Comment:
I would change this to "Cannot build a new version since no new CSVs have been uploaded."